web/modules/custom/collection/src/Plugin/Block/CollectionContentBlock.php

```diff
@@ -125,4 +130,11 @@ public function getCacheMaxAge() {
     return 0;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function blockAccess(AccountInterface $account) {
+    return ($this->currentRouteMatch->getRouteName() == 'entity.rdf_entity.canonical') ? AccessResult::allowed() : AccessResult::forbidden();
+  }
```

Comment:
Yes but now that the context is expanded, I only want this block to be shown for the canonical view of the rdf entity. And since it is a temporary block, I don't want to bother updating the configuration file with paths etc. Directly in the code and block it for now.